features/environment.py

```diff
@@ -182,20 +193,12 @@ def create_uat_lxd_image(context: Context, series: str) -> None:
     """
 
     if context.reuse_container:
```

Comment:
I think we can consolidate this line with the if series in context.reuse_container with a single line now
```suggestion
    if series in context.reuse_container:
```